src/services/kratos/index.ts

```diff
@@ -0,0 +1,194 @@
+import { getKratosConfig, getKratosMasterPhonePassword } from "@config"
+import {
+  AuthenticationKratosError,
+  IncompatibleSchemaUpgradeError,
+  KratosError,
+  LikelyNoUserWithThisPhoneExistError,
+  LikelyUserAlreadyExistError,
+  UnknownKratosError,
+} from "@domain/users/errors"
+import {
+  AdminUpdateIdentityBody,
+  Configuration,
+  Identity,
+  Session,
+  SuccessfulSelfServiceLoginWithoutBrowser,
+  SuccessfulSelfServiceRegistrationWithoutBrowser,
+  V0alpha2Api,
+  V0alpha2ApiInterface,
+} from "@ory/client"
+
+import { AxiosResponse } from "node_modules/@ory/client/node_modules/axios/index"
```

Comment:
will this hard coded path also work when embedded in the image?

Reply:
yup. but I think @dolcalmi said there is a better way to do it with resolutions? not sure, but at least this works.